nixos/modules/services/mail/maddy.nix

```
@@ -178,11 +186,15 @@ in {
           Define list of allowed domains.
         '';
       };
+
       config = mkOption {
         type = with types; nullOr lines;
         default = defaultConfig;
         description = ''
-          Server configuration.
+          Server configuration, see https://maddy.email for more information.
+          The default configuration of this module will setup a minimal maddy
+          instance for mail transfer without TLS encryption. This should
+          not be used in a production environment.
```

```suggestion
          instance for mail transfer without TLS encryption.
          
          <note><para>
          This should not be used in a production environment.
          </para></note>
```

Let's make that stand out.